config/vufind/Folio.ini

```diff
@@ -146,6 +146,11 @@ extraHoldFields = requiredByDate:pickUpLocation
 ; Regex mode example
 ;excludeHoldLocations[] = "/.*RESERVE.*/i"
 
+; Hide the place hold/recall/page link when an item is currently loaned to a
+; member of one of the following FOLIO patron groups.  For example, block requests
+; for items currently loaned out to other institutions.
```

Comment:
Should you explicitly note that we're using the name rather than the ID here? And might it be more robust to use the ID instead, or at least support the option? (I do understand there are advantages to readability, and we use names in other places like the filtering of holdings locations, so I'm not necessarily objecting to the current approach -- just thinking aloud).

Reply:
I agree on clarifying the match point; just added a comment.  I don't think I want to add support for ID since I modeled this off excludeHoldLocations, which doesn't support location ID.  I do get that theID is more stable; the FOLIO admin may change the name field, messing up the config.  But I think if we offer both options, in practice most people will still use name as it's more readable in the config.  But if you feel strongly, I can add ID support.

Reply:
I'm happy to leave it as a possible future enhancement, as long as we build things in a way that leaves the possibility open.